src/validationHelper.js

```diff
@@ -14,6 +13,20 @@ function validateField(rules, value) {
                 if (isNaN(value)) {
                     errors.push('only_number_field')
                 }
+                break;
+            case 'minimum':
+                rules.map((rule) => {
+                    if (value < rule.min_value) {
+                        errors.push('minimum_field')
+                    }
+                });
```

Comment:
We are already mapping over the rules. There's no need to map over them again.
I expect something like this:
``` javascript
switch(rule) {
    case 'required':
        // validate if present
        break;
    case 'string':
        // validate if string
        if (rules.includes('min')) {
            // validate string length
        }
    case 'number':
        // validate if number
        if (rules.includes('min')) {
            // validate number minimum
        }
}
```